### PR TITLE
Updates for Data Engineers

### PR DESCRIPTION
**Pull request by evantahler**
Outstanding Questions:

- [x] Do we remove `solutions/customer-data-platform` entirely?

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -48,13 +48,12 @@ export default function About({ pageProps }) {
...
-              and maintain. Funded by top-tier inventors, Grouparoo is creating
-              open-core applications to simplify and automate the data syncing
-              process within companies - solving both technical and
-              organizational challenges.
+              Our aim is to solve this problem by making it as simple as
+              deploying your website to change how your data is synced. Funded
+              by top-tier inventors, Grouparoo is creating open-core
+              applications to simplify and automate the data syncing process
+              within companies - solving both technical and organizational
+              challenges.
```
Comment (evantahler) [inline pages/about.tsx:56]: Thoughts on this new company description?  The old version talked about business users...

**Comment by andyjih** [comment]
> Outstanding Questions:
> 
> * [ ]  Do we remove `solutions/customer-data-platform` entirely?

Yeah, let's remove it.

File: pages/about.tsx (hunk 1 above)
Comment (andyjih) [reply pages/about.tsx:56]: Works for me!

File: pages/index.tsx [hunk 2]
```diff
@@ -140,28 +135,48 @@ export default function IndexPage({ pageProps, setReleaseNote }) {
...
-                <br />
-                <small>
-                  <Link href="/solutions/reverse-etl">
-                    <a>Learn more about reverse ETL</a>
-                  </Link>
-                </small>
+
+              <br />
+              <p style={{ textAlign: "left" }}>
+                We are at the next step in the data-driven evolution of how
```
Comment (andyjih) [inline pages/index.tsx]: I prefer the existing copy for this part of the page. This new description is quite wordy. I'd rather we add this copy further down on the page or elsewhere.

File: pages/index.tsx [hunk 3]
```diff
@@ -140,28 +135,48 @@ export default function IndexPage({ pageProps, setReleaseNote }) {
...
               />
+
+              <br />
+              <br />
+
+              <p style={{ textAlign: "left" }}>
+                Grouparoo helps these companies scale by making the integrations
+                easy. And it does it through open source to make sure that
+                flexibility, privacy, and low cost are at the heart of the
+                solution.
```
Comment (andyjih) [inline pages/index.tsx]:
```suggestion
                Grouparoo helps these companies scale by making integrations
                easy. And Grouparoo is open source, so flexibility and privacy are 
                at the heart of the solution.
```

File: pages/index.tsx [hunk 4]
```diff
@@ -430,6 +449,16 @@ export default function IndexPage({ pageProps, setReleaseNote }) {
...
               <br />
+              <p>
+                Grouparoo is powered by{" "}
+                <Link href="/docs/config">
+                  <a>code</a>
+                </Link>
+                . That means that just like the rest of your stack, you can
+                develop locally, commit your changes, and deploy. Our web-based
+                user interface makes it easy to generate configuration files for
+                even the most complex schema.
```
Comment (andyjih) [inline pages/index.tsx]:
```suggestion
                . You can use Grouparoo just like the rest of your tech stack; you can
                develop locally, commit your changes, and deploy. Our web-based
                user interface makes it easy to generate configuration files for
                even the most complex schema.
```

File: pages/solutions/census-alternative.mdx [hunk 5]
```diff
@@ -69,9 +69,9 @@ Census is a SaaS-based data syncing tool that started in 2018.
...
 
-### 🚀 Reverse ETL
+### 🚀 Reverse ETL powered by code
 
-Census is a Reverse ETL tool, and so is Grouparoo, but Grouparoo is so much more. Census requires that all of your data already be normalized in a formal data warehouse, and that you have a team to manage it. Grouparoo works great with data warehouses, but we also know that your business is still growing, and not all of your data has been brought into your data warehouse and is regularly updated. What about your customer support data in Zendesk? What about your NPS scores or onboarding surveys? Grouparoo’s library of sources is ready to help everyone, from the data warehouse engineer to the scrappy growth engineer, use their data, wherever it lives.
+Both Grouparoo and Census are [Reverse ETL](/solutions/reverse-etl) tools, but only Grouparoo was designed to fit into your existing engineering workflow. Grouparoo is powered by configuration files that you can generate and test locally, and then deploy with git - just like you deploy DBT projects today. Being open-source, Grouparoo can be run within your firewall or in our hosted cloud.
```
Comment (andyjih) [inline pages/solutions/census-alternative.mdx:74]: 👍

File: pages/solutions/segment-alternative.mdx [hunk 6]
```diff
@@ -101,6 +101,10 @@ Events are often a pain for engineers as well. When product engineers are buildi
   browser="true"
 />
 
+### 🚀 <span style={{ width:".5em", display:"inline-block" }} />Reverse ETL powered by code
+
+Both Grouparoo is a [Reverse ETL](/solutions/reverse-etl) tool, designed to fit into your existing engineering workflow. Grouparoo is powered by configuration files that you can generate and test locally, and then deploy with git. Being open-source, Grouparoo can be run within your firewall or in our hosted cloud.
```
Comment (andyjih) [inline pages/solutions/segment-alternative.mdx]:
```suggestion
Grouparoo is a [Reverse ETL](/solutions/reverse-etl) tool, designed to fit into your existing engineering workflow. Grouparoo is powered by configuration files that you can generate and test locally, and then deploy with git. Being open-source, Grouparoo can be run within your firewall or in our hosted cloud.
```

**Review by andyjih — CHANGES_REQUESTED** [review]
Nice! Added a few edits and suggestions.

File: pages/solutions/reverse-etl.mdx [hunk 7]
```diff
@@ -18,7 +18,7 @@ features:
     },
     {
       title: "📈  Improve advertising impact",
-      text: "With a Reverse ETL, you can automate list creation for effective advertising campaigns -- saving you advertising dollars and time.",
+      text: "With a Reverse ETL, you can automate list creation for effective advertising campaigns - saving you advertising dollars and time.",
```
Comment (andyjih) [inline pages/solutions/reverse-etl.mdx]:
```suggestion
      text: "With Reverse ETL, you can automate list creation for effective advertising campaigns - saving you advertising dollars and time.",
```

**Review by andyjih — APPROVED** [review]

**Pull request merged**